src/server/models/Player.js

```diff
@@ -113,6 +117,16 @@ export default class Player extends Model {
     this.index = index;
   }
 
+  gameLog(message) {
```

Comment:
why is this in `Player`, why not just modify `Game`'s `log` function

Reply:
The logging was originally player based including the padding was player based. I moved the padding to game but probably should have moved the logging there as well.